README.md

```diff
@@ -108,6 +108,7 @@ Transactions are a paged resource.  The response object will contain properties
 * `cursor`: An API cursor to fetch a specific set of results
 * `start`: An ISO-8601 datetime string to filter only transactions after `start`
 * `end`: An ISO-8601 datetime string to filter only transactions before `end`
+* `time_field`: Time field to filter by: created_date (default) or modified_date
```

Comment:
I'm assuming `time_field` affects what `start` and `end` refer to? Might be nice to make that a bit more explicit here.

Reply:
Updated